py/server/deephaven_internal/jvm/__init__.py

```diff
@@ -66,3 +66,25 @@ def init_jvm(*args, **kwargs):
             "Unable to initialize JVM, try setting the environment variable JAVA_HOME (JDK 11+ required)") from e
     ready()
     return result
+
+def init_py():
+    """Finishes starting Python to be usable from inside of a Java process. Not intended to be called in cases
+    where there process was start as Python, and Java was started from inside Python.
+    """
+
+    import jpy
+    import os
+    import sys
+    from deephaven_internal.stream import TeeStream
+
+    # Set stdin to /dev/null to prevent functions (like help()) that attempt to read from stdin from hanging python
+    # execution from within Java.
+    os.dup2(os.open("/dev/null", os.O_RDONLY), 0)
+
+    jpy.VerboseExceptions.enabled = True
+    # If you want jpy to tell you about all that it is doing, change this
+    # jpy.diag.flags = jpy.diag.F_ALL
```

Comment:
Should this be an ENV controlled option to help us debug users?

Reply:
That seems like a decent idea, but in a different PR (i'm just moving the contents of this file so we have one fewer way to screw up startup).